### PR TITLE
add vb language version 15 to command line parser

### PR DESCRIPTION
**Pull request by TyOverby**
Sister-pr to https://github.com/dotnet/roslyn/pull/11660

@dotnet/roslyn-compiler

### CODE REVIEW

**Comment by AdamSpeight2008** [comment]
#11666

**Comment by AlekseyTs** [comment]
LGTM

**Comment by TyOverby** [comment]
Thanks @AlekseyTs!  

@dotnet/roslyn-compiler: one more review please!  It's small, I promise!

File: src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb [hunk 1]
```diff
@@ -100,7 +100,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Dim parseDocumentationComments As Boolean = False ' Don't just null check documentationFileName because we want to do this even if the file name is invalid.
             Dim outputKind As OutputKind = OutputKind.ConsoleApplication
             Dim ssVersion As SubsystemVersion = SubsystemVersion.None
-            Dim languageVersion As LanguageVersion = LanguageVersion.VisualBasic14
+            Dim languageVersion As LanguageVersion = languageVersion.VisualBasic15
```
Comment (cston) [inline src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb]: **L**anguageVersion.VisualBasic15

**Comment by cston** [comment]
LGTM

File: src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb (hunk 1 above)
Comment (TyOverby) [inline src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb]: Fixed, thank you!

Comment (AdamSpeight2008) [inline src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb]: it's not like VB is case-sensitive. :smile:

Comment (TyOverby) [inline src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb]: lAnGuAgEvErSiOn it is then!

**Pull request merged**